debug no longer works with browserify

Any version of debug that I use past version 3.1.0 gives this error when trying to browserify code that includes this module

```
Cannot find module './common' from '[redacted]/node_modules/debug/dist'
      at FSReqWrap.oncomplete (fs.js:154:21)
```

There was a mention in #603 about a browserify incompatibility, but it doesn't appear to be the same issue.

Deep requiring into a specific file in the package from an unpkg url is not exactly a "works in the browser" test that feels like it was made in good faith.  Here's a barebones "try debug with browserify" that breaks (assumes you have npm@6 so that npx is a command you can run):

```sh
$ npm init -y
$ npm install debug@latest browserify@latest
$ echo "var log = require('debug')('example')" >example.js
$ npx browserify example.js
```
```
Error: Cannot find module './common' from '/usr/local/src/gar/projects/example/node_modules/debug/dist'
    at /usr/local/src/gar/projects/example/node_modules/browser-resolve/node_modules/resolve/lib/async.js:55:21
    at load (/usr/local/src/gar/projects/example/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/usr/local/src/gar/projects/example/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /usr/local/src/gar/projects/example/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:154:21)
```

--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,7 @@ dist: dist/debug.js dist/test.js
 .INTERMEDIATE: dist/debug.es6.js
 dist/debug.es6.js: src/*.js
 	@mkdir -p dist
-	browserify --standalone debug $< > $@
+	browserify --standalone debug . > $@
 
 dist/debug.js: dist/debug.es6.js
 	@mkdir -p dist

--- a/package.json
+++ b/package.json
@@ -45,5 +45,6 @@
     "xo": "^0.23.0"
   },
   "main": "./src/index.js",
-  "browser": "./dist/debug.js"
+  "browser": "./src/browser.js",
+  "unpkg": "./dist/debug.js"
 }

